contrib/experimental/great_expectations_experimental/expectations/expect_queried_slowly_changing_table_to_have_no_gaps.py

```diff
@@ -205,14 +204,14 @@ def _validate(
                         "threshold": 0.1,
                     },
                     "out": {"success": False},
-                    "only_for": ["sqlite"],
                 },
             ],
             "test_backends": [
                 {
                     "backend": "sqlalchemy",
                     "dialects": ["sqlite"],
-                }
+                },
```

Comment:
Can you delete the whole `test_backends` section, then add `"suppress_test_for": ["postgresql", "bigquery"],` after line 179

```
--- a/contrib/experimental/great_expectations_experimental/expectations/expect_queried_slowly_changing_table_to_have_no_gaps.py
+++ b/contrib/experimental/great_expectations_experimental/expectations/expect_queried_slowly_changing_table_to_have_no_gaps.py
@@ -177,6 +177,7 @@ class ExpectQueriedSlowlyChangingTableToHaveNoGaps(QueryExpectation):
                     },
                 },
             ],
+            "suppress_test_for": ["postgresql", "bigquery"],
             "tests": [
                 {
                     "title": "basic_positive_test",
@@ -206,13 +207,6 @@ class ExpectQueriedSlowlyChangingTableToHaveNoGaps(QueryExpectation):
                     "out": {"success": False},
                 },
             ],
-            "test_backends": [
-                {
-                    "backend": "sqlalchemy",
-                    "dialects": ["sqlite"],
-                },
-                {"backend": "spark", "dialects": None},
-            ],
         },
     ]
```